homeassistant/components/homeassistant/triggers/time.py

```diff
@@ -56,6 +63,7 @@ async def async_attach_trigger(
 ) -> CALLBACK_TYPE:
     """Listen for state changes based on configuration."""
     trigger_data = trigger_info["trigger_data"]
+    variables = trigger_info["variables"] or {}
```

Comment:
This would never be used right? So we could just set the dict directly on the `render_complex` call?

Reply:
I only added the access at the top of the method because everything else was accessing the trigger_info there.  I can access it in place.  

I just followed the path of trigger info, looks safe to just access it in place without the or {}.  Template.async_render accounts for None.  Will move.